dev/linters/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("@yamllint//:requirements.bzl", "all_requirements")
+load("@rules_python//python:defs.bzl", "py_binary")
+
+py_binary(
+    name = "yamllint",
+    srcs = ["//dev/linters:run_yamllint.py"],
+    data = [
+        # "//dev/linters:setup.cfg",
+    ],
+    main = "run_yamllint.py",
+    python_version = "PY2",
```

Comment:
We should probably be using Python3 throughout instead? I don't think we have any other Python dependencies at the moment, and Python 2 is going to EOL "soon". (And yamllint supports Python 3, as far as I can tell.)

Reply:
+1

Both Python 2 and 3 are available on the Docker image (along with pip). I'll drop Python 2 as soon as it reaches EOL.